release_notes/master-docinfo.xml

```diff
@@ -10,7 +10,7 @@
     <orgname>Red Hat Customer Content Services</orgname>
 </authorgroup>
 <legalnotice lang="en-US" version="5.0" xmlns="http://docbook.org/ns/docbook">
-  <para> 		Copyright <trademark class="copyright"></trademark> 2021 Red Hat, Inc. 	</para>
+  <para> 		Copyright <trademark class="copyright"></trademark> 2022 Red Hat, Inc. 	</para>
```

Comment:
Shouldn't we use {cdate} here, as you did in other places?

Reply:
> Shouldn't we use {cdate} here, as you did in other places?

Yes. This is already duplicated in downstream and merged.  I am trying to make sure the upstream and downstream always match.  I will fix it next time, because it is working now.